Cargo.toml

```diff
@@ -1,28 +1,35 @@
 [package]
-name = "kornia_rs"
-version = "0.0.8"
+name = "kornia-rs"
+version = "0.0.1"
```

Comment:
shouldn't we go to `0.1.0` or even to `1.0.0`, otherwise the versioning will be weird

Reply:
0.1.0 maybe